api/src/queries.js

```diff
@@ -0,0 +1,24 @@
+const items = {
+  get: `{
+    query(func: has(code)) @filter(eq(code, $code)) {	
+      code
+      description
+    }
+  }`,
+  search: `{
+    query(func: has(code)) @filter(allofterms(description, "$search")) {
```

Comment:
Some extra quotes here (using a hardcoded string needs quotes but vars don't, apparently)
```suggestion
    query(func: has(code)) @filter(allofterms(description, $search)) {
```